server/controllers/inventory/depots/index.js

```diff
@@ -156,6 +156,15 @@ function list(req, res, next) {
     .done();
 }
 
+function hasUuids(uuids, filters) {
```

Comment:
If you don't have this function, what happens?  If you just put in 
```js
filters.equals('uuids', 'uuid', 'd', true)
```
doesn't that work?  I thought it should.

Reply:
An array can have several items but our filter class doesn't add the corresponding number of question marks <b>?</b>


Maybe we should update ```filters.equals```

Reply:
It should add an SQL `IN ()` clause instead of an equals.  Isn't that what you need?

Reply:
In can see very well that the number of ```  '?'``` doesn't change if we have an array.
We can comeback to this issue in another PR I think.